### PR TITLE
Fix: Update Schema for ariaQuestion (fixes #190)

### LINKED ISSUE #190: Normalize ariaQuestion field across question components
The ariaQuestion field needs to be added to all question components.

### PR DESCRIPTION
**Pull request by joe-replin**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* Adds ariaQuestion field to modern schema standards

[//]: # (List appropriate steps for testing if needed)
### Testing
1. Install RP and test in AAT

### CODE REVIEW

**Comment by joe-replin** [comment]
Fixes #190

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: schema/component.schema.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
         "ariaQuestion": {
           "type": "string",
           "title": "ariaQuestion",
+          "description": "This will be read out by screen readers instead of reading the displayTitle, body & instruction fields when focusing on the options. To be clear and concise, ensure the text encompasses only the question associated.",
```
Comment (oliverfoster) [inline schema/component.schema.json:39]: display title is two words in the AAT. we don't references the json names directly, but the attribute name from the content.schema.json

Comment (joe-replin) [reply schema/component.schema.json]: Good call out. Updated.

**Review by oliverfoster — APPROVED** [review]

**Review by joe-allen-89 — APPROVED** [review]

**Pull request merged**